azure-pipelines.yml

```diff
@@ -213,3 +213,38 @@ jobs:
       continueOnError: true
       condition: not(succeeded())
 
+- job: Correctness_Rebuild
```

Comment:
Do you want to add this to README badges?

Reply:
Eventually 😄